type/__nextcloud/manifest

```diff
@@ -27,6 +27,26 @@ version_ge() {
     }'; return $?
 }
 
+# Version compare function if the major version is too different (only upwards)
+#
+# Arguments:
+#  1: curr version
+#  2: new version
+#
+# Return code:
+#  0: major difference bigger than 1
+#  1: major not bigger than 1
+version_majdiff() {
+    printf "%s" "$1" | awk -F '[^0-9.]' -v target="$2" '
+    BEGIN {
+        getline
+        split($1, x, ".")
+        split(target, y, ".")
+        if((y[1] - x[1]) > 1) exit 0
+        else exit 1
+    }'; return $?
```

Comment:
Please don't `return $?`. In shell a function will always implicitly return the exit code of the last command it executed.